spec/javascripts/Stats/lib/application_details.spec.ts

```diff
@@ -6,12 +6,14 @@ it('should return a weird object', () => {
     application: {
       id: 11,
       name: 'My Application',
+      link: '/p/admin/applications/11',
```

Comment:
Hahaha, the name of the test... (not added by this PR)
"it('should return a weird object'" :laughing:

Reply:
Ooooh I didn't even see that 🤣 
I'm having mixed feelings about changing it... `should return an object` or `should return an object with application details` just don't look as good.

Reply:
Nah, let's keep it :wink:

Reply:
I have to disagree. The object is only weird if you don't know what's it about, once you know what kind of object it is, the object is not weird anymore, so the test doesn't match the expectation of returning a weird object and therefore it should fail. Since this test is not deterministic, as its results depend on the mental frame of the reader, I suggest to rename it to `'should return a known object unless you don't know what's this about; should return a weird object otherwise'`

Reply:
I have no words to show how much I appreciate this last comment so I'll add a GIF to better express myself:
![](https://media.giphy.com/media/5hHOBKJ8lw9OM/giphy.gif)

Thank you VERY much @jlledom